src/js/hooks/cycleCount/useCountStep.jsx

```diff
@@ -327,6 +327,7 @@ const useCountStep = () => {
       }
       await submitCount();
     } finally {
+      dispatch(eraseDraft());
```

Comment:
what if submit fails? We probably don't want to erase the draft since it's still in progress.

Not an essential change, but is there any easy way to only erase the draft if we actually succeed in submitting the count?

Reply:
yeah, you are right. Actually you would still have the access to the data if you don't refresh the page, but to be safe, I moved it to the `try` clause, so that the draft is only erased if we successfully submit.